cloud-custodian-cron/examples/example-values.yaml

```diff
@@ -0,0 +1,59 @@
+image:
+  repository: cloudcustodian/c7n
+  pullPolicy: IfNotPresent
+
+scheduledPolicies: 
+- name: daily-policies
+  concurrencyPolicy: {}
+  schedule: "* * * * *"
```

Comment:
we probably want both of these schedules to line up to their names (daily, weekly) these look like their every minute atm.